src/Ordering.Infrastructure/EntityConfigurations/OrderItemEntityTypeConfiguration.cs

```diff
@@ -15,23 +15,23 @@ public void Configure(EntityTypeBuilder<OrderItem> orderItemConfiguration)
         orderItemConfiguration.Property<int>("OrderId");
 
         orderItemConfiguration
-            .Property("_discount")
+            .Property(o => o.Discount)
```

Comment:
FYI now that these are regular properties, doing HasColumnName is no longer needed and can be removed.